api/scripts/tcr/tcr.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+alteredFilePaths=$(git ls-files -m '*.js');
+files=$@
+
+function lint () {
+  npx eslint $alteredFilePaths --fix
+}
+
+function test () {
+  npm run test:api:path $files -- --bail
+}
+
+function commit () {
+  echo "TCR => COMMIT"
+  git commit -am "TCR:WIP"
+}
+
+function revert () {
+  echo "TCR => REVERT"
+  git checkout HEAD lib/
+}
+
+if [ ! -z "$alteredFilePaths" ] ; then
+  if lint ; then
+    test && commit || revert
```

Comment:
Si les tests passent et que `commit` échoue, ça va jouer `revert`, il vaut mieux :
```suggestion
    if test ; then
      commit
    else
      revert
    fi
```

Plus d'infos :
- https://github.com/koalaman/shellcheck/wiki/SC2015

Reply:
Done.